resources/TrialApp.didact.md

```diff
@@ -30,39 +30,39 @@ For example, change line 26 from: `<h1> Welcome to Azure! </h1>` to this: `<h1>
 
 ### Step 3: Deploy your changes
 
-1. Select the blue up arrow icon to deploy your code to your Trial App.
+1. Select the blue up arrow icon to deploy your code to your trial app.
 
 <img src="./trialAppTutorial/deployButtonScreenshot.png" style="width: 256px;">
 
-2. Select the folder containing the Trial App code you cloned and modified.
+2. Select the folder containing the trial app code you cloned and modified.
 
 <img src="./trialAppTutorial/selectFolderToDeployScreenshot.png" style="width: 512px;">
 
-3. Select your Trial App.
+3. Select your trial app.
 
 <img src="./trialAppTutorial/selectTrialAppDeployScreenshot.png" style="width: 512px;">
 
 4. You will see the following notification once your deployment is complete. You can click `Browse` to view the changes live.
 
 <img src="./trialAppTutorial/deployCompleteScreenshot.png" style="width: 512px;">
 
-You can also browse your website by right-clicking your Trial app and selecting `Browse`.
+You can also browse your website by right-clicking your trial app and selecting `Browse`.
 
 <img src="./trialAppTutorial/browseWebsiteRedScreenshot.png" style="width: 256px;">
 
 ### Step 4: Transfer your App to an Azure Subscription
```

Comment:
It doesn't look like you're giving the steps title-casing treatment so should make app lower-case.